src/components/dictionaryConcepts/containers/CreateConcept.jsx

```diff
@@ -60,7 +59,8 @@ export class CreateConcept extends Component {
       datatype: 'None',
       names: [],
       descriptions: [],
-      answers: [],
+      // answers: [],
```

Comment:
@AGMETEOR , kindly remove the commented code.

Reply:
@Duncanian I have removed it.